types/messaging.ts

```diff
@@ -1,9 +1,37 @@
-// TODO: This needs expanding
-interface MessageTrigger {
-  /** Name of the trigger */
-  id: string;
-  /** Additional parameters of the trigger */
-  params?: Array<string|number>
+import { TriggerAction } from "./triggerAction";
+import { MessagingAction } from "./specialMessageAction";
+
+type FluentId = { string_id: string };
```

Comment:
Do we allow `attributes` in every `FluentOrString`? I guess not

Reply:
No this is only for CFRs where `accesskey` is needed.